### DISCUSSION TITLE
[Docs] Add all CSS Variables

### DISCUSSION THREAD

**Discussion by r-priyam** [ElMassimo/iles#21] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
To add all the CSS variables used in generating the static site

**Additional context**
CSS variables like - https://v2.vuepress.vuejs.org/reference/default-theme/styles.html#palette-file

**Comment by ElMassimo** [comment, marked as answer]
@r-priyam Unlike Vitepress and Vuepress, îles does not provide a default theme.

You have full control on which CSS you import (from Vue components, or from `src/app.ts`).

Once the APIs have stabilized, I will document how to create plugins and themes for îles. I will also consider creating an official `@islands/docs` preset to create documentation sites (although [Vitepress is already a good option for that](https://iles-docs.netlify.app/faqs#vitepress)).

**Reply by r-priyam** [reply]
Aye Aye, Got it. I would love to see more of this because I guess there are so many features around. Loved iles, Thank you for the fast response.
